### DISCUSSION TITLE
No RPI-GPIO after pacman -Syu

### DISCUSSION THREAD

**Discussion by Slaughthammer** [rern/rAudio#742] [Fixed Issues] [answered]
I just set up my system and then after I updated with pacman -Syu my previously correctly working python script failed returning:
`Traceback (most recent call last):
  File "/usr/local/bin/sleeptimer.py", line 7, in <module>
    import RPi.GPIO as GPIO
ModuleNotFoundError: No module named 'RPi'`

Any fix for this?

**Comment by rern** [comment, marked as answer]
`RPi.GPIO` has been not good when `Python` upgraded to newer versions. rAudio uses `WiringPi` instead.
At the moment, `RPi.GPIO` cannot run on `Python 3.10`.

**Reply by Slaughthammer** [reply]
Thanks! That does work. However, I need to install python-pip first so I can pip install WiringPi. Tough goings for the occasional Linuxer...

**Reply by rern** [reply]
There's no need to install anything. `WiringPi` is already installed.
Anyway, `RPi.GPIO` in +R repo has been upgraded so try `pacman -Syu` again.

**Reply by Slaughthammer** [reply]
I can confirm that this solves the problem.
